src/main/java/org/mskcc/cbio/oncokb/service/SlackService.java

```diff
@@ -161,8 +164,13 @@ private TextObject getTextObject(String title, String content) {
         // Add user info section
         blocks.add(buildUserInfoBlock(user));
 
-        // Add Approve button
-        blocks.add(buildPlainTextBlock(ACADEMIC_CLARIFICATION_NOTE));
+        if(withClarificationNote) {
+            // Add clarification note
+            blocks.add(buildPlainTextBlock(ACADEMIC_CLARIFICATION_NOTE));
+        } else {
+            // Add Approve button
+            blocks.add(buildPlainTextBlock(COMMERCIAL_APPROVE_NOTE));
```

Comment:
Why we add commercial approve note in academic blocks

Reply:
@victoria34 updated, was a typo above causing the mistake.